mkosi/log.py

```diff
@@ -14,13 +14,17 @@
 LEVEL = 0
 
 
+def terminal_is_dumb() -> bool:
+    return not sys.stdout.isatty() and not sys.stderr.isatty() and os.getenv("TERM", "") != "dumb"
```

Comment:
Wouldn't this make more sense
```suggestion
    return not sys.stdout.isatty() or not sys.stderr.isatty() or os.getenv("TERM", "") == "dumb"
```
since if `TERM=dumb` we should probably hide colour even if stderr and stdout are attached to a TTY, also if either stderr or stdout are not a TTY assuming a dumb terminal might be safer.